usr/event_poll.c

```diff
@@ -214,7 +214,7 @@ void event_loop(struct iscsi_ipc *ipc, int control_fd, int mgmt_ipc_fd)
 						  "exiting", errno);
 					break;
 				} else {
-					log_debug(1, "Poll was woken by an alarm");
+					log_debug(4, "Poll was woken by an alarm");
```

Comment:
So that's at least two debug log messages that have changed the level at which they print, i.e. to only print at a higher debug level. That's fine, but I think that should be mentioned in the description, as long as you are updating it (since it needs to be more clear about the "log level" being the "-d" debug log level of iscsid.